packages/ui/src/ui/store/selectors/global/index.ts

```diff
@@ -204,19 +204,23 @@ export const getAllIdmGroupNamesSorted = createSelector(
 export const getGlobalShowLoginDialog = (state: RootState) => {
     if (state.global.authWay) {
         return (
-            getConfigData().allowLoginDialog &&
+            getConfigData().allowPasswordAuth &&
             state.global.ytAuthCluster &&
             (!state.global.login || state.global.showLoginDialog)
         );
     }
 
     return (
-        getConfigData().allowLoginDialog && (!state.global.login || state.global.showLoginDialog)
+        getConfigData().allowPasswordAuth && (!state.global.login || state.global.showLoginDialog)
     );
 };
 
 export const getAuthPagesEnabled = () => {
-    return getConfigData().allowLoginDialog;
+    return getConfigData().allowPasswordAuth;
+};
+
+export const getUserManagementEnabled = () => {
```

Comment:
We should not allow to use it for regular users, it should be enabled only for admins:
```ts
import {isDeveloper} from '../../../store/selectors/global/is-developer';
...
export const getUserManagementEnabled = createSelector([isDeveloper], (isAdmin) => {
    return isAdmin && getConfigData().allowPasswordAuth;
});
```